docs/document/content/user-manual/shardingsphere-proxy/startup/docker.cn.md

```diff
@@ -26,8 +26,12 @@ docker pull apache/shardingsphere-proxy
 ```bash
 git clone https://github.com/apache/shardingsphere
 ./mvnw clean install
-cd shardingsphere-distribution/shardingsphere-proxy-distribution
-./mvnw clean package -Prelease,docker
+cd distribution/proxy
+../../mvnw clean package -Prelease,docker
+```
+为加快构建速度，可用如下命令代替```./mvnw clean install```：
+```
+./mvnw clean install -Prelease -T16C -DskipTests -Djacoco.skip=true -Dcheckstyle.skip=true -Drat.skip=true -Dmaven.javadoc.skip=true -B
```

Comment:
T16C will not accelerate the build process, especially on github ubuntu-latest. you could try it on a test environment.
BTW, release process does not support concurrency build.